docker/local-universe/add-stub-universe.sh

```diff
@@ -47,14 +54,14 @@ jq --arg PACKAGE "$PACKAGE" '.packages[] | select(.name == $PACKAGE)' $FILE > st
 
 cat stub-repo/$PACKAGE.json | jq '.config'  > stub-repo/packages/${LETTER}/${PACKAGE}/0/config.json
 cat stub-repo/$PACKAGE.json | jq '.resource' > stub-repo/packages/${LETTER}/${PACKAGE}/0/resource.json
-cat stub-repo/$PACKAGE.json | jq '.command' > stub-repo/packages/${LETTER}/${PACKAGE}/0/command.json
+[[ $(cat stub-repo/$PACKAGE.json | jq '.command') = "null"  ]] || cat stub-repo/$PACKAGE.json | jq '.command' > stub-repo/packages/${LETTER}/${PACKAGE}/0/command.json
 cat stub-repo/$PACKAGE.json | jq 'del(.command, .marathon, .resource, .config, .releaseVersion)' > stub-repo/packages/${LETTER}/${PACKAGE}/0/package.json
-cat stub-repo/$PACKAGE.json | jq -r '.marathon.v2AppMustacheTemplate' | base64 -D > stub-repo/packages/${LETTER}/${PACKAGE}/0/marathon.json.mustache
+cat stub-repo/$PACKAGE.json | jq -r '.marathon.v2AppMustacheTemplate' | ${BASE64} > stub-repo/packages/${LETTER}/${PACKAGE}/0/marathon.json.mustache
 
 done
 
 echo ""
 echo "Full stub-repo contents:"
 ls -alh stub-repo/packages/*/*/*
 
-echo ""
+echo ""
```

Comment:
Add a newline.

Reply:
He's on 🌴  so may not get to see this for a bit

Reply:
Sounds good. Merge it when you are ready.